### ISSUE TITLE
`CocoaLumberjack` build issues

### ISSUE THREAD

**Issue by lxalfonso** [SVGKit/SVGKit#775]
### New Issue Checklist

* [✅] I have read and understood the [CONTRIBUTING guide](https://github.com/SVGKit/SVGKit/blob/2.x/CONTRIBUTING.md)
* [✅] I have reproduced the problem while running in Xcode, and read the Console log, and acted upon any relevant error or warning messages
* [N/A] I have reproduced the problem while running in Xcode's Debugger, and used breakpoints to examine the actual values of variables vs the expected values
* [ ✅] I am not using CocoaPods, or: I am using CocoaPods, but have verified my podspec points to SVGKit v2.x

### Issue Description and Steps

Hey everyone! I was wondering if anyone's still had the `CocoaLumberjack` minimum version issue that has been brought up here before. My project's using SPM and currently I get either one of two errors:
A. If my dependency is set on **Version: Up to Next Major `3.0.0`**, then I see: 
```
The package product 'CocoaLumberjack' requires minimum platform version 11.0 for the iOS platform, but this target supports 9.0
```
B. If my dependency is set on the branch `3.x`, then I see:
```
'CocoaLumberjack/CocoaLumberjack.h' file not found
```

Not sure if I'm missing something, but has anyone encountered anything similar?

**Comment by bstillitano** [comment]
Same issue for me on branch `3.x`

**Comment by bubudrc** [comment]
same issue here..

**Comment by airspeed** [comment]
Be sure to import `CocoaLumberjackSwift` as well:
https://github.com/CocoaLumberjack/CocoaLumberjack#swift-package-manager

**Comment by bubudrc** [comment]
but... if you set, on the Package file the dependency, this should be automatically added, and that it's what happened. The error is other

**Comment by lxalfonso** [comment]
> Be sure to import `CocoaLumberjackSwift` as well:
> https://github.com/CocoaLumberjack/CocoaLumberjack#swift-package-manager

@airspeed Are you referring to something that we would be able to do inside of our own projects? The error I'm referring to is coming from the SVGKit source code, and in an Objective-C file at that. I'm not sure if `CocoaLumberjackSwift` applies here.

**Comment by airspeed** [comment]
@bubudrc @lxalfonso I needed to extra import `CocoaLumberjack` as Package in *my* project, and from the list of modules I selected both `CocoaLumberjack` and `CocoaLumberjackSwift`.
This *in addition* to `SVGKit` importing `CocoaLumberjack`. Now my projects builds.

**Comment by lxalfonso** [comment]
@airspeed 
Doesn't seem to be working for my project. I actually already did have `CocoaLumberjack` and the Swift variant added to my project as a dependency as I use it log other areas of my code. Still no resolution for this on my end sadly. But thank you for the response!

**Comment by bstillitano** [comment]
I’ve created a fork of this library that removes the cocoalumberjack dependency https://github.com/bstillitano/SVGKit

From: Alexander Alfonso ***@***.***>
Date: Tuesday, 24 January 2023 at 8:15 am
To: SVGKit/SVGKit ***@***.***>
Cc: Brandon Stillitano ***@***.***>, Comment ***@***.***>
Subject: Re: [SVGKit/SVGKit] `CocoaLumberjack` build issues (Issue #775)

@airspeed<https://github.com/airspeed>
Sadly doesn't seem to be working for my project. I actually already did have CocoaLumberjack and the Swift variant added to my project as a dependency as I use it log other areas of my code. Still no resolution for this on my end sadly.

**Comment by bubudrc** [comment]
I'm using this PR version https://github.com/SVGKit/SVGKit/pull/764 that removed CocoaLumberjack dependency, and it's working great

**Comment by lxalfonso** [comment]
Fixed in PR #780
